pkg/expr/service_test.go

```diff
@@ -138,7 +138,7 @@ func TestDSQueryError(t *testing.T) {
 		cfg:          setting.NewCfg(),
 		dataService:  me,
 		pCtxProvider: pCtxProvider,
-		features:     &featuremgmt.FeatureManager{},
+		features:     featuremgmt.TestFeatureToggles(),
```

Comment:
```suggestion
		features:     featuremgmt.WithFeatures(),
```
WithFeatures is used in many of the other tests -- it lets you stick in features if you want them enabled.  I don't think we need another test implementation (maybe an alias?)

Reply:
Ah, thanks @ryantxu - I missed this. I'll replace it!